### PR TITLE
add tags, add jar name, update codebuild project naming and desc

### PR DESCRIPTION
**Pull request by radengoots**

### CODE REVIEW

File: examples/simple/outputs.tf [hunk 1]
```diff
@@ -1,4 +1,4 @@
 output "codebuild_project" {
   description = "CodeBuild project name"
```
Comment (andysaputra) [inline examples/simple/outputs.tf]: Change to "The name of CodeBuild project"

File: examples/simple/main.tf [hunk 2]
```diff
@@ -2,9 +2,10 @@ provider "aws" {
   region  = "ap-southeast-1"
 }
 
-module "codebuild_backend_common_rules-engine" {
+module "codebuild_backend_common_example" {
   source          = "../../"
-  source_location = "https://github.com/traveloka/repo-location.git"
-  project_path    = "backend/common/rules-engine"
+  product_domain  = "your-product-domain"
```
Comment (andysaputra) [inline examples/simple/main.tf]: use real product_domain, i.e. bei

File: variables.tf [hunk 3]
```diff
@@ -1,10 +1,15 @@
+variable "product_domain" {
+  type        = "string"
+  description = "The owner of the postgres proxy."
```
Comment (andysaputra) [inline variables.tf]: remove dot (.)

File: variables.tf [hunk 4]
```diff
@@ -1,10 +1,15 @@
...
+}
+
 variable "source_location" {
-  description = "Link to github repository"
+  description = "Link to github repository."
```
Comment (andysaputra) [inline variables.tf]: remove dot (.)

File: variables.tf [hunk 5]
```diff
@@ -1,10 +1,15 @@
...
   type        = "string"
 }
 
 variable "project_path" {
-  description = "Relative path to the project"
+  description = "Relative path to the project."
```
Comment (andysaputra) [inline variables.tf]: remove dot (.)

File: examples/simple/main.tf (hunk 2 above)
Comment (radengoots) [reply examples/simple/main.tf]: is it ok to put it on public domain?

File: main.tf [hunk 6]
```diff
@@ -18,4 +18,10 @@ resource "aws_codebuild_project" "this" {
     location  = "${var.source_location}"
     buildspec = "${data.template_file.buildspec.rendered}"
   }
+
+  tags {
+    "ProductDomain" = "${var.product_domain}"
```
[the remark this answers is not in the document]
Comment (radengoots) [reply main.tf]: add as jar_name

File: main.tf [hunk 7]
```diff
@@ -18,4 +18,11 @@ resource "aws_codebuild_project" "this" {
...
+    "JarName"   = "${var.jar_name}"
```
Comment (andysaputra) [inline main.tf]: reformat

File: variables.tf [hunk 8]
```diff
@@ -1,16 +1,31 @@
...
+  type        = "string"
+  description = "Binary Owner"
+}
+
+variable "jar_name" {
+  type        = "string"
+  description = "Jar name, e.g. rules-engine-repository-db-proxy"
+}
+
+variable "library_name" {
```
Comment (andysaputra) [inline variables.tf]: is this used?

File: main.tf [hunk 9]
```diff
@@ -1,6 +1,6 @@
 resource "aws_codebuild_project" "this" {
-  name         = "${replace("TvSQLPublisher_${var.project_path}", "/", "_")}"
-  description  = "CodeBuild project for publishing ${var.project_path} proxy binary"
+  name         = "TvSQLPublisher_${var.product_domain}_${var.jar_name}"
```
Comment (andysaputra) [inline main.tf]: I think it's better to remove tvSqlPublisher, and change underscore to hyphens

**Pull request merged**